Makefile

```diff
@@ -70,3 +70,9 @@ generate: ## Genenerate proto files and server and client stubs from OpenAPI spe
 
 generate-mocks: ## Regenerate all needed mocks, in order to add new mocks generation add //go:generate to file from witch mocks should be generated
 	$(GOGEN) ./...
+
+integration: integration-metrics
```

Comment:
should the target be `integration-tests` to be consistent with the `unit-tests` target?